docs/content/Caching/Caching.md

```diff
@@ -33,56 +33,44 @@ the query hasn't changed, the cached value will be returned. Otherwise, a
-
-- The result of the `refreshKey` SQL query is different from the previous one. At this stage `refreshKey` won't be refreshed in foreground if it's available in the cache.
-- The query cache entry has expired. The default expiration time for cubes with a default `refreshKey` is 6 hours, cubes specifying their own expire in 24 hours.
-
-### Refresh Key Implementation
+```js
+cube(`Orders`, {
+  // This refreshKey tells Cube.js to refresh data every 5 minutes
+  refreshKey: {
+    every: `5 minute`
+  }
 
-In order for Cube.js to properly expire cache entries and refresh in the background, Cube.js needs a value to track through time. There's a built in default `refreshKey` query strategy that works the following way:
+  // With this refreshKey Cube.js will only refresh the data if
+  // the value of previous MAX(created_at) changed
+  // By default Cube.js will check this refreshKey every 10 seconds
+  refreshKey: {
+    sql: `SELECT MAX(created_at) FROM orders`
+  }
+});
+```
 
-1. Check used pre-aggregations for query and use [pre-aggregations refreshKey](pre-aggregations#refresh-key), if no pre-aggregations are used then…
-2. Check the `max` of time dimensions with `updated` in the name, if none exist then…
-3. Check the `max` of any existing time dimension, if none exist then…
-4. Check the row count for this cube.
+By default, Cube.js will refresh the cache in the background in the [development mode][link-development-mode]. When development mode is disabled you need to explicitly activate background refresh by setting the corresponding environment variable `CUBEJS_SCHEDULED_REFRESH_TIMER=true`.
 
-You can set up a custom refresh check SQL by changing [refreshKey](cube#parameters-refresh-key) property in a cube's Data Schema. There are situations where the default strategy doesn't work, such as:
+ We recommend to run background refresh in the separate worker for production deployments. Please consult the [production checklist][link-production-checklist] for more information.
 
- - forecasting data or other time series data where the timestamps being queried are always the same or always purely dependent on the query
- - non-time series data like lists of customers or list of suppliers that doesn't have a time dimension
- - other data that may only get `UPDATE`s and few `INSERT`s, meaning the total row count doesn't change very often.
 
-In these instances, Cube.js needs a query crafted to detect updates to the rows that power the cubes. Often, a `MAX(updated_at_timestamp)` for OLTP data will accomplish this, or examining a metadata table for whatever system is managing the data to see when it last ran.
+[link-development-mode]: /configuration/overview#development-mode
+[link-production-checklist-refresh]: /deployment/production-checklist#set-up-refresh-worker
 
+If the background refresh is disabled Cube.js will refresh the cache during the
+query time. Since it could lead to delays in the query processing for end users
```

Comment:
```suggestion
-query time. Since it could lead to delays in the query processing for end users
+execution. Since this could lead to delays in responding to end-users, we
```